Apps/W1/Intrastat/app/src/IntrastatRepCustTemCard.PageExt.al

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.Inventory.Intrastat;
+
+using Microsoft.Sales.Customer;
+
+pageextension 4817 "Intrastat Rep. Cust. Tem. Card" extends "Customer Templ. Card"
+{
+    layout
+    {
+        addafter(Shipping)
```

Comment:
I would suggest to use addlast instead of addafter since Shipping is the last group,